### PR TITLE
Create modern signs for SR2020

### PR DESCRIPTION
**Pull request by tpoomlmly**
Actual sign content coming soon.

### CODE REVIEW

**Comment by RealOrangeOne** [comment]
@tpoomlmly these signs look great! :tada: 

I do wonder about the fact they're saved as both PNG and PDF. Does PowerPoint allow you to output slides as individual SVGs or alike? If not personally I think it'd be more useful to keep the PDF versions rather than PNG. 

Alternatively, we could review and merge this PR as-is, and deal with formatting issues later? :thinking:

**Comment by RealOrangeOne** [comment]
@TomWheal as you created the signs last year, would be good to have your review too!

**Review by TomWheal — APPROVED** [review]
The Powerpoint  looks identical to the one I created which is great!

Im not sure if having it in binaries like PDFs and PNGs and PPTX is a good idea, you can save the PowerPoint as an XML which retains all the Microsoft functionality, but allows for better sub versioning and also allows to be opened on office. 

Also printing from a bunch of PDFs can be annoying, I found it easy to print from the PowerPoint and just copy the slides I wanted multiple of. 

Only small comments as long as people are happy with having a bunch of binaries in the repo, I'm happy for this to be merged.

**Comment by tpoomlmly** [comment]
I've added SVGs of all the signs I've made and converted the powerpoint files to zips with xml files inside.

**Comment by PeterJCLaw** [comment]
These signs look great, thanks for preparing them.

Some thoughts on the format discussion:
- We seem to have the logo in here several times, including both SVG and PNG files. It would be great if we didn't have these duplicated. Maybe consider submoduling in the https://github.com/srobo/brand repo so we don't have the logos duplicated. That should also mean that future updates are easier -- just need to bump the submodule rather than copy/paste them. We've been through this for our website repos (which now use the brand repo as submodules), so it would be great to avoid introducing another case we'll need to either track or fix later. (If we do do this, please consider rebasing to remove the large binaries we don't need in the history)
- Please can we avoid having lots of build outputs as binaries in git repos. I can see an argument for having a single PowerPoint template here (though I'd personally rather we stuck with SVGs as they play _much_ nicer with git and our other tooling, while also not needing proprietary software to be opened correctly†) though I think having PDFs is overkill when we have SVGs. That said, the SVGs here aren't exactly usable (they look like they've been minified) so it might be worth having _just_ the PDFs. (If possible, build outputs should be built by CI, as we do elsewhere, so they're reproducible and associated with versions, but not actually in the repo)
- Either way if we're going to have build outputs in the repo (and again I'd rather we didn't), let's have only a single format of them (and rebase this branch to remove the rest so the history isn't absurdly huge).

In summary:
- I think it's important to minimise the amount of ~binaries~ build outputs (PDFs, SVGs-from-PowerPoint) we have here, though I accept that some may be inevitable
- once we have minimised the binaries, please let's rebase (before merging) so they're not in the history and so those which remain are only present once

† Yes I know LibreOffice can technically open them, though it doesn't always work properly. If we're going to do binaries let's use the LibreOffice formats so they're reproducible.
